pkg/engine/response/response.go

```diff
@@ -102,6 +102,9 @@ type RuleResponse struct {
 	// JSON patches, for mutation rules
 	Patches [][]byte `json:"patches,omitempty"`
 
+	// Resource generated by the generate rules of a policy
+	GeneratedResource unstructured.Unstructured
```

Comment:
should we add `json` tag here for `GeneratedResource` field `json:"genereatedResource,omitempty"`

Reply:
done